index.js

```diff
@@ -1,17 +1,51 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items.sort((a, b) => a - b);
+    this.length = this.items.length;
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (pos >= 0 && pos < this.length) {
+      return this.items[pos];
+    } else {
+      throw new OutOfBoundsException("Position is out of bounds");
+    }
+  }
 
-  max() {}
+  max() {
+    if (this.length === 0) {
+    throw new EmptySortedList("Empty SortedList");
+    }
+    return this.items[this.length -1];
```

Comment:
Tambien aqui lo que podemos hacer es utilizar la función max() retorna el valor máximo presente en la lista. Para hacer esto 🔥 , utiliza el método Math.max(...this.items), que toma una cantidad variable de argumentos y devuelve el valor más grande entre ellos. En este caso, el operador de propagación ... se utiliza para pasar los elementos individuales de la lista como argumentos a la función Math.max().
```
 return Math.max(...this.items)
 
 ```